### PR TITLE
🐛 FIX: Avoid importlib.metadata

### PR DESCRIPTION
**Pull request by Bemte**
- Fixed some typos (not sure about the websocket, please check Discord message)
- worked on issue 13
- test for __version__ is only testing for the date currently

### CODE REVIEW

**Comment by Bemte** [comment]
Ah, well, seems the version string is formatted differently in these automated tests as opposed to my local machine, so I manged to fail them all; go me! :D

File: smalld/smalld.py [hunk 1]
```diff
@@ -2,19 +2,27 @@
...
 
 from .standard_listeners import add_standard_listeners
 
+
+try:
+    from importlib.metadata import version
+    __version__ = version("smalld")
+except ImportError:
+    from pkg_resources import get_distribution
+    __version__ = get_distribution("smalld").version
```
Comment (ianagbip1oti) [inline smalld/smalld.py:18]: We can just use the pkg_resources version, because that'll work on py3.8 too

File: test/test_version.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from smalld import __version__
+from datetime import datetime
+
+# tests that the version matches the date
+def test_version():
+    today = datetime.now().strftime("%Y%m%d")
+    assert __version__.split('d')[-1] == today
```
Comment (ianagbip1oti) [inline test/test_version.py:7]: Just an `assert __version__` should be sufficient, to make sure it's not None

File: setup.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
     install_requires=[
         "attrdict>=2.0.1",
         "requests>=2.23.0",
-        "websocket_client>=0.57.0",
+        "websocket-client>=0.57.0",
```
Comment (ianagbip1oti) [inline setup.py:15]: Heh, so apparently websocket-client or websocket_client should work. So, if websocket-client is working with conda for you but websocket_client isn't. I'm fine with leaving this.

**Pull request closed without merging**